examples/src/components/CopyLink.tsx

```diff
@@ -1,34 +1,28 @@
 'use client'
+import { ArrowUpRightIcon } from '@heroicons/react/24/outline'
 
-import { useState } from 'react'
-import { LinkIcon } from '@heroicons/react/24/outline'
+import Title from '@/components/Title'
 
 export default function CopyLink() {
-  const [copied, setCopied] = useState(false)
-
-  const copyLinkToClipboard = () => {
-    const currentPageURL = window.location.href
-
-    navigator.clipboard
-      .writeText(currentPageURL)
-      .then(() => {
-        setCopied(true)
-      })
-      .catch((error) => {
-        console.error('Failed to copy link: ', error)
-      })
-  }
-
   return (
-    <div className="pt-6 flex text-left items-center text-neutral-500">
-      <span className="pr-2">Share this document</span>
-      <button
-        className="flex text-sm items-center gap-1 px-2 py-1 rounded-lg bg-neutral-50 border-white border transition-all hover:bg-white"
-        onClick={copyLinkToClipboard}
-      >
-        <LinkIcon className="h-3 w-3" />
-        {copied ? 'Copied!' : 'Copy Link'}
-      </button>
+    <div className="mr-2 mb-2 text-left items-center text-neutral-500 border-2 border-yellow-200 rounded-lg bg-yellow-50 px-6 py-3">
+      <div className="flex justify-between items-center pb-1">
+        <Title>Collaborate on this document</Title>
+        <div className="flex">
+          <a
+            className="text-sm flex items-center gap-1 px-3 py-1 rounded-lg bg-pink-950 text-white border transition-all "
+            href={window.location.href}
```

Comment:
It looks like this is failing to compile causing [the y-sweet demos test to fail](https://vercel.com/drifting-corp/y-sweet-demos/88R41ACSZu6LAHmqBGP54pRjGDo1).

I think we can get around using `window.location.href` by using Nextjs's `usePathname` hook to get the path of the current page.